### ISSUE TITLE
Idea to enforce the use of alt attributes

### ISSUE THREAD

**Issue by robdekort** [statamic/ideas#496]
Clients rarely add ALT attributes. Now there are drastic tricks like not showing the image if it doesn't have an alt text, but maybe we can be try to enforce it in the asset browser.

I was a small tweak first. How about a column `Alt text` with a green checkmark if you got it, and a red cross if you don't. 

This could be extend to the appearance of the asset field.

**Comment by stefankempf** [comment]
I would also have two other suggestions:

1. show an input field during initial upload, where the alt text can be stored. Then at least, the user sees the alt text input once.
2. have an option to make an alt text required. If a user tries to publish an article containing images without alt text, show an error warning (for example: "alt text not set") like the usual required warning. Maybe add a link to open the image detail stack, where the alt text can be set.

**Comment by AndreasSchantl** [comment]
Totally agree. In the mean time you could take a look at the widget from the Peak starter kit. It shows all the images that don't have an alt text yet.

**Comment by Alpenverein** [comment]
We are running in simular problems aswell.. We use different required fields in our assets' blueprint. While required fields are not validated during upload the image was available to users even without copyright or legal information.
